sys/vm/vm_page.h

```diff
@@ -1011,6 +1073,5 @@ vm_page_domain(vm_page_t m)
 	return (0);
 #endif
 }
-
```

Comment:
Seems unrelated?

Reply:
Arg, this one seems to have sneaked back in. ><